src/Components/OATPropertyEditor/Internal/EditorPropertiesTab/EditorPropertiesTab.tsx

```diff
@@ -0,0 +1,212 @@
+                const updateModel = () => {
+                    oatPageDispatch({
+                        type: OatPageContextActionType.UPDATE_REFERENCE,
+                        payload: {
+                            modelId: parentModelId,
+                            reference: selectedItemCopy
+                        }
+                    });
+                };
+
+                const undoUpdate = () => {
+                    oatPageDispatch({
+                        type: OatPageContextActionType.GENERAL_UNDO,
+                        payload: {
+                            models: oatPageState.currentOntologyModels,
+                            positions:
+                                oatPageState.currentOntologyModelPositions,
+                            selection: oatPageState.selection
+                        }
+                    });
+                };
+
+                execute(updateModel, undoUpdate);
+            }
+        },
+        [
+            execute,
+            oatPageDispatch,
+            oatPageState.currentOntologyModelPositions,
+            oatPageState.currentOntologyModels,
+            oatPageState.selection,
+            parentModelId,
+            propertiesKeyName,
+            selectedItem
+        ]
+    );
+
+    // side effects
+
+    // styles
+    const propertyInspectorStyles = getPropertyInspectorStyles();
+    const classNames = getClassNames(styles, {
+        theme: useExtendedTheme()
+    });
+
+    return (
+        <Stack className={classNames.root}>
+            <Stack.Item>
+                <PropertiesModelSummary
+                    isSupportedModelType={isSupportedModelType}
+                    selectedItem={selectedItem}
+                />
+            </Stack.Item>
+            <Stack.Item>
+                <div className={propertyInspectorStyles.propertyListHeaderWrap}>
+                    <Stack className={propertyInspectorStyles.rowSpaceBetween}>
+                        <Label>{`${t('OATPropertyEditor.properties')} ${
+                            propertyList.length > 0
+                                ? `(${propertyList.length})`
+                                : ''
+                        }`}</Label>
+                        {isSupportedModelType && (
+                            <PropertyTypePicker onSelect={onAddType} />
+                        )}
+                    </Stack>
+                </div>
+            </Stack.Item>
+
+            <Stack.Item
+                grow
+                styles={classNames.subComponentStyles.propertyListStack}
+            >
+                {isDTDLReference(selectedItem) || isDTDLModel(selectedItem) ? (
+                    <PropertyList
+                        selectedItem={selectedItem}
+                        properties={propertyList}
+                        parentModelId={parentModelId}
+                    />
+                ) : (
+                    'Property list not supported'
```

Comment:
Should this be in a translation call?

Reply:
yes, yes it should.

Reply:
![image](https://user-images.githubusercontent.com/57726991/215563303-4fd56d67-3c29-4a41-a755-74f22b64758e.png)

Reply:
Done.